fem/tmop.cpp

```diff
@@ -2798,8 +2798,6 @@ void TMOP_Integrator::GetSurfaceFittingErrors(double &err_avg, double &err_max)
          loc_sum += std::abs((*surf_fit_gf)(i));
       }
    }
-   err_avg = loc_sum / loc_cnt;
-   err_max = loc_max;
```

Comment:
@SchmidtMat can you remind me why was this change needed?

Some tests are currently failing, because when MFEM is built in parallel, but a serial test is run, `err_avg` and  `err_max` remain empty, due to the `return` on 2803.

Reply:
The old version failed in parallel in the case when there was no surface on a processor local subdomain. In this case it err_avg = loc_sum / loc_cnt; is a division by 0.

Reply:
I see, thanks. I'll propose a small PR to fix those.

Reply:
Thanks. Sorry I missed this one.